Description: update paths for xen-4.6 in tests
 This was done using grep + sed
 I recommend simply re-generating automatically at next merge.
 One way to do so would be:
 $ dquilt push ubuntu/fix-xen-xml-in-tests.patch
 # if it (likely) fails to apply, create a new one on all files in test/
 $ dquilt delete -r ubuntu/fix-xen-xml-in-tests.patch
 $ dquilt new ubuntu/fix-xen-xml-in-tests.patch
 $ dquilt add $(grep -rl 'xen\/boot\/hvmloader' tests/)
 $ for f in $(grep -rl 'xen\/boot\/hvmloader' tests/); do sed -i 's/xen\/boot\/hvmloader/xen-4.6\/boot\/hvmloader/g' ${f}; done
 $ dquilt refresh
 # then copy over this header
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: no

--- a/tests/domaincapsschemadata/libxl-xenfv-usb.xml
+++ b/tests/domaincapsschemadata/libxl-xenfv-usb.xml
@@ -6,7 +6,7 @@
   <vcpu max='128'/>
   <os supported='yes'>
     <loader supported='yes'>
-      <value>/usr/lib/xen/boot/hvmloader</value>
+      <value>/usr/lib/xen-4.6/boot/hvmloader</value>
       <value>/usr/lib/xen/boot/ovmf.bin</value>
       <enum name='type'>
         <value>rom</value>
--- a/tests/domaincapsschemadata/libxl-xenfv.xml
+++ b/tests/domaincapsschemadata/libxl-xenfv.xml
@@ -6,7 +6,7 @@
   <vcpu max='128'/>
   <os supported='yes'>
     <loader supported='yes'>
-      <value>/usr/lib/xen/boot/hvmloader</value>
+      <value>/usr/lib/xen-4.6/boot/hvmloader</value>
       <value>/usr/lib/xen/boot/ovmf.bin</value>
       <enum name='type'>
         <value>rom</value>
--- a/tests/domaincapstest.c
+++ b/tests/domaincapstest.c
@@ -248,7 +248,7 @@
 
     if (VIR_ALLOC(firmwares[0]) < 0 || VIR_ALLOC(firmwares[1]) < 0)
         goto cleanup;
-    if (VIR_STRDUP(firmwares[0]->name, "/usr/lib/xen/boot/hvmloader") < 0 ||
+    if (VIR_STRDUP(firmwares[0]->name, "/usr/lib/xen-4.6/boot/hvmloader") < 0 ||
         VIR_STRDUP(firmwares[1]->name, "/usr/lib/xen/boot/ovmf.bin") < 0)
         goto cleanup;
 
--- a/tests/sexpr2xmldata/sexpr2xml-fv-autoport.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-autoport.sexpr
@@ -21,7 +21,7 @@
     (online_vcpus 1)
     (image
         (hvm
-            (kernel /usr/lib/xen/boot/hvmloader)
+            (kernel /usr/lib/xen-4.6/boot/hvmloader)
             (boot d)
             (device_model /usr/lib/xen/bin/qemu-dm)
             (keymap en-us)
--- a/tests/sexpr2xmldata/sexpr2xml-fv-autoport.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-autoport.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (loader /usr/lib/xen/boot/hvmloader)(kernel '')\
+(image (hvm (loader /usr/lib/xen-4.6/boot/hvmloader)(kernel '')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot d)(cdrom '/root/boot.iso')\
 (acpi 1)(vnc 1)(keymap ja)))(device (vbd (dev 'hda')\
 (uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(hpet 1)(vnc 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(hpet 0)(vnc 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-kernel.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-kernel.sexpr
@@ -4,6 +4,6 @@
 (image (hvm (kernel '/var/lib/xen/vmlinuz.2Dn2YT')\
 (ramdisk '/var/lib/xen/initrd.img.0u-Vhq')\
 (args ' method=http://download.fedora.devel.redhat.com/pub/fedora/linux/\
-core/test/5.91/x86_64/os  ')(loader '/usr/lib/xen/boot/hvmloader')(vcpus 2)\
+core/test/5.91/x86_64/os  ')(loader '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 2)\
 (usb 1)(serial pty)))(device (vbd (dev 'ioemu:xvda')\
 (uname 'file:/root/some.img')(mode 'w'))))
--- a/tests/sexpr2xmldata/sexpr2xml-fv-kernel.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-kernel.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>2</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <kernel>/var/lib/xen/vmlinuz.2Dn2YT</kernel>
     <initrd>/var/lib/xen/initrd.img.0u-Vhq</initrd>
     <cmdline> method=http://download.fedora.devel.redhat.com/pub/fedora/linux/core/test/5.91/x86_64/os  </cmdline>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-legacy-vfb.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-legacy-vfb.sexpr
@@ -21,7 +21,7 @@
     (online_vcpus 1)
     (image
         (hvm
-            (kernel /usr/lib/xen/boot/hvmloader)
+            (kernel /usr/lib/xen-4.6/boot/hvmloader)
             (boot c)
             (device_model /usr/lib/xen/bin/qemu-dm)
             (keymap en-us)
--- a/tests/sexpr2xmldata/sexpr2xml-fv-legacy-vfb.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-legacy-vfb.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-localtime.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-localtime.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(vnc 1)(keymap ja)(localtime 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-localtime.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-localtime.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(vnc 1)(keymap ja)))(device (vbd (dev 'ioemu:hda')\
 (uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel tcp:localhost:9999)\
 (serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8ff')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (cdrom '/root/boot.iso')(acpi 1)(usb 1)(parallel none)\
 (serial (/dev/ttyS0 /dev/ttyS1))(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)))(device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8ff')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial (none /dev/ttyS1))(device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial file:/tmp/serial.log)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)))(device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial null)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial pipe:/tmp/serial.pipe)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)))(device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial pty)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial stdio)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial telnet:localhost:9999,server,nowait)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial tcp:localhost:9999,server,nowait)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial udp:localhost:9998@localhost:9999)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 1)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial unix:/tmp/serial.sock,server,nowait)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(vnc 1)(keymap ja)(soundhw 'all')))(device (vbd (dev 'ioemu:hda')\
 (uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-sound.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-sound.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(vnc 1)(keymap ja)(soundhw 'sb16,es1370')))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-sound.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-sound.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(usbdevice mouse)(vnc 1)(keymap ja)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(usb 1)(usbdevice tablet)(vnc 1)(keymap ja)))\
 (device (vbd (dev 'ioemu:hda')(uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-utc.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-utc.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)(cdrom '/root/boot.iso')\
 (acpi 1)(vnc 1)(keymap ja)))(device (vbd (dev 'ioemu:hda')\
 (uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-utc.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-utc.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv-v2.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-v2.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)(acpi 1)(vnc 1)\
 (keymap ja)))(device (vbd (dev 'hda:disk')(uname 'file:/root/foo.img')\
 (mode 'w')))(device (vbd (dev 'hdc:cdrom')(uname 'file:/root/boot.iso')\
--- a/tests/sexpr2xmldata/sexpr2xml-fv-v2.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-v2.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-fv.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-fv.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(vnc 1)(keymap ja)))(device (vbd (dev 'ioemu:hda')\
 (uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-fv.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.sexpr
@@ -22,7 +22,7 @@
     (online_vcpus 1)
     (image
         (hvm
-            (kernel /usr/lib/xen/boot/hvmloader)
+            (kernel /usr/lib/xen-4.6/boot/hvmloader)
             (vnc 1)
             (vncunused 1)
             (device_model /usr/lib/xen/bin/qemu-dm)
--- a/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/sexpr2xmldata/sexpr2xml-vif-rate.sexpr
+++ b/tests/sexpr2xmldata/sexpr2xml-vif-rate.sexpr
@@ -1,7 +1,7 @@
 (domain (domid 3)(name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd275cdaca517769660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(boot c)\
 (acpi 1)(vnc 1)(keymap ja)))(device (vbd (dev 'ioemu:hda')\
 (uname 'file:/root/foo.img')(mode 'w')))\
--- a/tests/sexpr2xmldata/sexpr2xml-vif-rate.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-vif-rate.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/testutilsxen.c
+++ b/tests/testutilsxen.c
@@ -94,7 +94,7 @@
     if ((guest = virCapabilitiesAddGuest(caps, VIR_DOMAIN_OSTYPE_HVM,
                                          VIR_ARCH_X86_64,
                                          "/usr/lib/xen/bin/qemu-system-i386",
-                                         "/usr/lib/xen/boot/hvmloader",
+                                         "/usr/lib/xen-4.6/boot/hvmloader",
                                          nmachines, machines)) == NULL)
         goto cleanup;
     machines = NULL;
--- a/tests/xlconfigdata/test-fullvirt-multiserial.xml
+++ b/tests/xlconfigdata/test-fullvirt-multiserial.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='x86_64' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-escape-paths.cfg
+++ b/tests/xmconfigdata/test-escape-paths.cfg
@@ -23,6 +23,6 @@
 serial = "none"
 soundhw = "sb16,es1370"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader&test"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader&test"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", """phy:/dev/HostVG/XenGuest'",hdb,w""", "file:/root/boot.iso&test,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-escape-paths.xml
+++ b/tests/xmconfigdata/test-escape-paths.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader&amp;test</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader&amp;test</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-default-feature.cfg
+++ b/tests/xmconfigdata/test-fullvirt-default-feature.cfg
@@ -4,7 +4,7 @@
 memory = 394
 vcpus = 1
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 hpet = 1
 viridian = 0
--- a/tests/xmconfigdata/test-fullvirt-default-feature.xml
+++ b/tests/xmconfigdata/test-fullvirt-default-feature.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-force-hpet.cfg
+++ b/tests/xmconfigdata/test-fullvirt-force-hpet.cfg
@@ -23,6 +23,6 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-force-hpet.xml
+++ b/tests/xmconfigdata/test-fullvirt-force-hpet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-force-nohpet.cfg
+++ b/tests/xmconfigdata/test-fullvirt-force-nohpet.cfg
@@ -23,6 +23,6 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-force-nohpet.xml
+++ b/tests/xmconfigdata/test-fullvirt-force-nohpet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-localtime.cfg
+++ b/tests/xmconfigdata/test-fullvirt-localtime.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-localtime.xml
+++ b/tests/xmconfigdata/test-fullvirt-localtime.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-net-netfront.cfg
+++ b/tests/xmconfigdata/test-fullvirt-net-netfront.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-net-netfront.xml
+++ b/tests/xmconfigdata/test-fullvirt-net-netfront.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-new-cdrom.cfg
+++ b/tests/xmconfigdata/test-fullvirt-new-cdrom.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-new-cdrom.xml
+++ b/tests/xmconfigdata/test-fullvirt-new-cdrom.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-nohap.cfg
+++ b/tests/xmconfigdata/test-fullvirt-nohap.cfg
@@ -23,6 +23,6 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-nohap.xml
+++ b/tests/xmconfigdata/test-fullvirt-nohap.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-parallel-tcp.cfg
+++ b/tests/xmconfigdata/test-fullvirt-parallel-tcp.cfg
@@ -22,6 +22,6 @@
 parallel = "tcp:127.0.0.1:7777"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-parallel-tcp.xml
+++ b/tests/xmconfigdata/test-fullvirt-parallel-tcp.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-file.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-file.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "file:/tmp/serial.log"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-file.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-file.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-null.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-null.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "null"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-null.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-null.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-pipe.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-pipe.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "pipe:/tmp/serial.pipe"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-pipe.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-pipe.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-pty.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-pty.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "pty"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-pty.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-pty.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-stdio.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-stdio.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "stdio"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-stdio.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-stdio.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "telnet:127.0.0.1:9999,server,nowait"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-tcp.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-tcp.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "tcp:127.0.0.1:7777"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-tcp.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-tcp.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-udp.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-udp.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "udp:127.0.0.1:9999@0.0.0.0:99998"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-udp.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-udp.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-serial-unix.cfg
+++ b/tests/xmconfigdata/test-fullvirt-serial-unix.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "unix:/tmp/serial.sock,server,nowait"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-serial-unix.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-unix.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-sound.cfg
+++ b/tests/xmconfigdata/test-fullvirt-sound.cfg
@@ -23,6 +23,6 @@
 serial = "none"
 soundhw = "sb16,es1370"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-sound.xml
+++ b/tests/xmconfigdata/test-fullvirt-sound.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-usbmouse.cfg
+++ b/tests/xmconfigdata/test-fullvirt-usbmouse.cfg
@@ -22,7 +22,7 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
 usb = 1
--- a/tests/xmconfigdata/test-fullvirt-usbmouse.xml
+++ b/tests/xmconfigdata/test-fullvirt-usbmouse.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-usbtablet-no-bus.xml
+++ b/tests/xmconfigdata/test-fullvirt-usbtablet-no-bus.xml
@@ -3,7 +3,7 @@
   <uuid>c7a5fdb2-cdaf-9455-926a-d65c16db1809</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <currentMemory unit='KiB'>403456</currentMemory>
--- a/tests/xmconfigdata/test-fullvirt-usbtablet.cfg
+++ b/tests/xmconfigdata/test-fullvirt-usbtablet.cfg
@@ -22,7 +22,7 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
 usb = 1
--- a/tests/xmconfigdata/test-fullvirt-usbtablet.xml
+++ b/tests/xmconfigdata/test-fullvirt-usbtablet.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-fullvirt-utc.cfg
+++ b/tests/xmconfigdata/test-fullvirt-utc.cfg
@@ -22,6 +22,6 @@
 parallel = "none"
 serial = "none"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "d"
 disk = [ "phy:/dev/HostVG/XenGuest2,hda,w", "file:/root/boot.iso,hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-fullvirt-utc.xml
+++ b/tests/xmconfigdata/test-fullvirt-utc.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-no-source-cdrom.cfg
+++ b/tests/xmconfigdata/test-no-source-cdrom.cfg
@@ -20,6 +20,6 @@
 parallel = "none"
 serial = "pty"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "c"
 disk = [ "phy:/dev/sda8,hda,w", ",hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-no-source-cdrom.xml
+++ b/tests/xmconfigdata/test-no-source-cdrom.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/xmconfigdata/test-pci-devs.cfg
+++ b/tests/xmconfigdata/test-pci-devs.cfg
@@ -21,6 +21,6 @@
 parallel = "none"
 serial = "pty"
 builder = "hvm"
-kernel = "/usr/lib/xen/boot/hvmloader"
+kernel = "/usr/lib/xen-4.6/boot/hvmloader"
 boot = "c"
 disk = [ "phy:/dev/sda8,hda,w", ",hdc:cdrom,r" ]
--- a/tests/xmconfigdata/test-pci-devs.xml
+++ b/tests/xmconfigdata/test-pci-devs.xml
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='i686' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <features>
--- a/tests/xml2sexprdata/xml2sexpr-escape.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-escape.sexpr
@@ -5,7 +5,7 @@
 (ramdisk '/var/lib/xen/initrd.img.0u-Vhq')\
 (args ' method=http://download.fedora.devel.redhat.com/pub/fedora/linux/\
 core/test/5.91/x86_64/os&version="devel"  ')\
-(loader '/usr/lib/xen/boot/hvmloader')(vcpus 2)(boot c)(usb 1)(parallel none)\
+(loader '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 2)(boot c)(usb 1)(parallel none)\
 (serial pty)(device_model '/usr/lib/xen/bin/qemu-dm')(rtc_timeoffset 0)\
 (localtime 0)))(localtime 0)\
 (device (vbd (dev 'xvda:disk')(uname 'file:/root/\'\\some.img')(mode 'w'))))
--- a/tests/xml2sexprdata/xml2sexpr-escape.xml
+++ b/tests/xml2sexprdata/xml2sexpr-escape.xml
@@ -6,7 +6,7 @@
     <kernel>/var/lib/xen/vmlinuz.2Dn2YT</kernel>
     <initrd>/var/lib/xen/initrd.img.0u-Vhq</initrd>
     <cmdline> method=http://download.fedora.devel.redhat.com/pub/fedora/linux/core/test/5.91/x86_64/os&amp;version="devel"  </cmdline>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
   </os>
   <memory unit='KiB'>430080</memory>
   <vcpu>2</vcpu>
--- a/tests/xml2sexprdata/xml2sexpr-fv-force-hpet.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-force-hpet.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(hpet 1)(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-force-hpet.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-force-hpet.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-force-nohpet.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-force-nohpet.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(hpet 0)(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-force-nohpet.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-force-nohpet.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-kernel.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-kernel.sexpr
@@ -4,7 +4,7 @@
 (image (hvm (kernel '/var/lib/xen/vmlinuz.2Dn2YT')\
 (ramdisk '/var/lib/xen/initrd.img.0u-Vhq')\
 (args ' method=http://download.fedora.devel.redhat.com/pub/fedora/linux/\
-core/test/5.91/x86_64/os  ')(loader '/usr/lib/xen/boot/hvmloader')(vcpus 2)\
+core/test/5.91/x86_64/os  ')(loader '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 2)\
 (boot c)(usb 1)(parallel none)(serial pty)\
 (device_model '/usr/lib/xen/bin/qemu-dm')\
 (rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-kernel.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-kernel.xml
@@ -6,7 +6,7 @@
     <kernel>/var/lib/xen/vmlinuz.2Dn2YT</kernel>
     <initrd>/var/lib/xen/initrd.img.0u-Vhq</initrd>
     <cmdline> method=http://download.fedora.devel.redhat.com/pub/fedora/linux/core/test/5.91/x86_64/os  </cmdline>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
   </os>
   <memory unit='KiB'>430080</memory>
   <vcpu>2</vcpu>
--- a/tests/xml2sexprdata/xml2sexpr-fv-localtime.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-localtime.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 1)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-localtime.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-localtime.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-net-netfront.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-net-netfront.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
 (usb 1)(parallel none)(serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)(vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
 (localtime 0)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-net-netfront.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-net-netfront.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-net-rate.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-net-rate.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
 (usb 1)(parallel none)(serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)(vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
 (localtime 0)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-net-rate.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-net-rate.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-parallel-tcp.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-parallel-tcp.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel tcp:localhost:9999)\
 (serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-parallel-tcp.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-parallel-tcp.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2-ports.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2-ports.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial (/dev/ttyS0 /dev/ttyS1))(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2-ports.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2-ports.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2nd-port.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2nd-port.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial (none /dev/ttyS1))(device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2nd-port.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-dev-2nd-port.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-file.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-file.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial file:/tmp/serial.log)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-file.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-file.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-null.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-null.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial null)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-null.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-null.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-pipe.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-pipe.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial pipe:/tmp/serial.pipe)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-pipe.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-pipe.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-pty.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-pty.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial pty)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-pty.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-pty.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-stdio.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-stdio.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial stdio)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-stdio.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-stdio.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp-telnet.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp-telnet.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial telnet:localhost:9999,server,nowait)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp-telnet.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp-telnet.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial tcp:localhost:9999,server,nowait)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-tcp.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-udp.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-udp.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial udp:localhost:9998@localhost:9999)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-udp.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-udp.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-unix.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-unix.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)\
 (serial unix:/tmp/serial.sock,server,nowait)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-serial-unix.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-serial-unix.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-sound.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-sound.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (soundhw 'sb16,es1370')(device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-sound.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-sound.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-usbmouse.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-usbmouse.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(usbdevice mouse)(parallel none)\
 (serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-usbmouse.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-usbmouse.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-usbtablet.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-usbtablet.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (cdrom '/root/boot.iso')(acpi 1)(usb 1)(usbdevice tablet)(parallel none)\
 (serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-usbtablet.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-usbtablet.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-utc.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-utc.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv-utc.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-utc.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv-v2.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-v2.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
 (usb 1)(parallel none)(serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)(vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
 (localtime 0)\
--- a/tests/xml2sexprdata/xml2sexpr-fv-vncunused.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv-vncunused.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
 (usb 1)(parallel none)(serial none)(device_model '/usr/lib64/xen/bin/qemu-dm')\
 (vnc 1)(vncunused 1)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
 (localtime 0)(device (vbd (dev 'hda:disk')\
--- a/tests/xml2sexprdata/xml2sexpr-fv-vncunused.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv-vncunused.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-fv.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-fv.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'fvtest')(memory 400)(maxmem 400)(vcpus 1)\
 (uuid 'b5d70dd2-75cd-aca5-1776-9660b059d8bc')(on_poweroff 'destroy')\
 (on_reboot 'restart')(on_crash 'restart')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)\
 (acpi 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib64/xen/bin/qemu-dm')(vnc 1)\
 (vncunused 0)(vncdisplay 17)(keymap 'ja')(rtc_timeoffset 0)(localtime 0)))\
--- a/tests/xml2sexprdata/xml2sexpr-fv.xml
+++ b/tests/xml2sexprdata/xml2sexpr-fv.xml
@@ -3,7 +3,7 @@
   <uuid>b5d70dd275cdaca517769660b059d8bc</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>409600</memory>
--- a/tests/xml2sexprdata/xml2sexpr-no-source-cdrom.sexpr
+++ b/tests/xml2sexprdata/xml2sexpr-no-source-cdrom.sexpr
@@ -1,7 +1,7 @@
 (vm (name 'test')(memory 350)(maxmem 382)(vcpus 1)\
 (uuid 'cc2315e7-d26a-307a-438c-6d188ec4c09c')(on_poweroff 'destroy')\
 (on_reboot 'destroy')(on_crash 'destroy')\
-(image (hvm (kernel '/usr/lib/xen/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
+(image (hvm (kernel '/usr/lib/xen-4.6/boot/hvmloader')(vcpus 1)(boot c)(acpi 1)\
 (apic 1)(pae 1)(usb 1)(parallel none)(serial none)\
 (device_model '/usr/lib/xen/bin/qemu-dm')(vnc 1)(vncunused 0)(vncdisplay 6)\
 (rtc_timeoffset 0)(localtime 0)))(localtime 0)\
--- a/tests/xml2sexprdata/xml2sexpr-no-source-cdrom.xml
+++ b/tests/xml2sexprdata/xml2sexpr-no-source-cdrom.xml
@@ -3,7 +3,7 @@
   <uuid>cc2315e7d26a307a438c6d188ec4c09c</uuid>
   <os>
     <type>hvm</type>
-    <loader>/usr/lib/xen/boot/hvmloader</loader>
+    <loader>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='hd'/>
   </os>
   <memory unit='KiB'>391168</memory>
